wandb/sdk/data_types/graph.py

```diff
@@ -249,20 +249,24 @@ class Graph(Media):
     represents the graph as an array of nodes and edges.  The nodes can have
     labels that can be visualized by wandb.
 
-    Examples:
-        Import a keras model:
-        ```
-        Graph.from_keras(keras_model)
-        ```
-
     Attributes:
         format (string): Format to help wandb display the graph nicely.
         nodes ([wandb.Node]): List of wandb.Nodes
         nodes_by_id (dict): dict of ids -> nodes
         edges ([(wandb.Node, wandb.Node)]): List of pairs of nodes interpreted as edges
         loaded (boolean): Flag to tell whether the graph is completely loaded
         root (wandb.Node): root node of the graph
-    """
+
+    Examples:
+
+    Import a keras model
+
+    ```python
+    import wandb
+
+    Graph.from_keras(keras_model)
+    ```
+    """  # noqa: D412
```

Comment:
Why disable this instead of fixing it?

Reply:
Ah. Oops. So I added D412 to allow missing blank line after last section. In the markdown, we need a space between "Examples" and the text.